src/modules/mavlink/mavlink_main.cpp

```diff
@@ -1620,26 +1621,21 @@ Mavlink::configure_streams_to_default(const char *configure_single_stream)
 		configure_stream_local("GLOBAL_POSITION_INT", 5.0f);
```

Comment:
Can we lower this one?

Reply:
I'd say yes if we can strip out the others. The messages are scaled back relative to the initial value, so having this slightly higher than the others is somewhat of a coping mechanism to get the data through at 1 or 2 Hz.